packages/sdk-bundles-api/src/replay/sdk-to-bundle/timeline.types.ts

```diff
@@ -1,8 +0,0 @@
-export interface ReplayTimelineEntry {
-  kind: string;
-  start: number;
-  end: number;
-  data: unknown;
-}
-
-export type ReplayTimelineData = ReplayTimelineEntry[];
```

Comment:
This was in the wrong folder since these messages are actually sent from the bundle to the SDK rather than the other way round.